src/config.js

```diff
@@ -1,8 +1,13 @@
+var fs = require('fs');
+var dir = __dirname + '/../../translations/';
```

Comment:
This doesn't make sense at all...

Reply:
```
require.main.filename
```
https://stackoverflow.com/a/20292957

But at the same time I'm not convinced that we need to install it outside node_modules, let's investigate why we can't install/update files in node_modules?